apps/builder/app/builder/features/props-panel/control.tsx

```diff
@@ -201,34 +188,15 @@ export function Control({
   onChangePropValue,
   setCssProperty,
 }: ControlProps) {
-  const meta = getComponentMetaProps(component);
+  const meta = getComponentMeta(component)?.props;
```

Comment:
It was intentionally separately to not bloat runtime with huge jsons.

Reply:
Ah, got it. I misunderstood because we still export/import these jsons from the `*.ws.tsx` files and not as a separate export. So they still were in all bundles.

Will:
- switch back to getComponentMetaProps()
- add a comment about intention
- try to export from `*.ws.tsx` separately

Reply:
Done